documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility-behaviors.md

```diff
@@ -36,10 +33,11 @@ public class ExampleBehaviorActionProvider : BehaviorActionProvider
                                       new PropertyAction("SourceName"),
                                       new PropertyAction("SourceObject"),
                                       new PropertyAction("ActiveOnFocus")));
-                                      
+        
+        //Specify MaxCount to limit number of times the user is allowed to add this behavior                              
```

Comment:
>number of times the user is allowed to add this behavior

number of times the user is allowed to perform this action?

Reply:
@koljaka I still don't understand what this means "the user is allowed to add this behavior".

Reply:
When BehaviorActionGroup is added to provider, it will add new item to ComboBox (on right side of 'Add Behavior' action). By adding this property we limit how many times this specific behavior could be added to XAML. Once added, item will be removed from ComboBox and will appear in list of existing behaviors. Once removed - it will appear again as separate item in ComboBox

Reply:
Why would we allow the same property to be added to XAML more than once?

Reply:
we are limiting Behavior not separate property of it. Check updated example - we create behavior editor with 3 properties. If we don't add MaxCount, then if you selected it in ComboBox and click twice 'Add Behavior' - two same behaviors will be added to XAML and two separate editors inside 'Suggested Actions' will be created for each of them and each editor will contains 3 properties.

Reply:
Thank you @koljaka, now I get it. :-) Perhaps we should put the MaxCount initialization on a new line to make this distinction cleaner.